### ISSUE TITLE
[Improvement] Change theme folder from /mx-themes/ to just /themes/

### ISSUE THREAD

**Issue by adamwintle** [modxcms/mxt#3]
This isn't a bug, just a suggestion open for discussion.  

To me It makes more sense to me to rename the /mx-themes/ folder to just /themes/ to avoid any confusion and to keep things simple.
- There currently aren't any other MODX theme systems (that I'm aware of) which would need to use a /theme/ folder and cause a conflict
- Makes it clear that this is _the_ themes folder, rather than potentially something else
- If someone wanted to use just the /themes/ folder its going to cause a problem because MODX isn't built to handle themes properly anyway; multiple themes systems is going to make things more complex.
- Using `mx-` uses three extra characters to type in files or on the command line

**Comment by dubrod** [comment]
while it does logically makes sense. we try to keep naming conventions exact b/c of the inherent Creative Freedom that is native to MODX we don't know what the folder `themes` could have been in the system before you installed MXT

**Comment by adamwintle** [comment]
Yeah I see what you mean; perhaps on setup the user could be prompted specify a theme folder, with the value pre-filled with a default `mx-themes`?

Problem with this is if they later want to install another theme (for interchangeability) the system would have to know to place the theme files into their custom specified theme folder.
